.github/workflows/hypersdk-unit-tests.yml

```diff
@@ -8,9 +8,11 @@ on:
     branches:
       - main
   pull_request:
+    types: [labeled,synchronize,reopened]
```

Comment:
May make sense to remove `labeled` (adding labels will interrupt all other label jobs).